src/Components/ClusterRules/ClusterRules.js

```diff
@@ -244,6 +274,12 @@ const ClusterRules = ({ cluster }) => {
       sortingRows = [...rows].sort((firstItem, secondItem) => {
         const fst = firstItem[0].rule[CLUSTER_RULES_COLUMNS_KEYS[index]];
         const snd = secondItem[0].rule[CLUSTER_RULES_COLUMNS_KEYS[index]];
+        if (index === 3) {
+          //sorting for the impacted column
+          const firstDate = new Date(firstItem[0].rule.impacted || 0);
+          const secondDate = new Date(secondItem[0].rule.impacted || 0);
+          return firstDate > secondDate ? d : secondDate > firstDate ? -d : 0;
```

Comment:
This is the same as for the line two lines below. What about declaring `fst` and `snd` with `let`, and use if-else clause with a condition `if (index === CLUSTER_RULES_IMPACTED_CELL)`, where you will then decide which values to assign to `fst` and `snd`?..

Reply:
I applied your suggestion. Thanks